src/amp-ads/50_AMP_Story_Ads/amp-story-auto-ads.html

```diff
@@ -0,0 +1,335 @@
+      color: white;
+    }
+    amp-story-page {
+      background: white;
+    }
+    amp-story h1 {
+      font-size: 46px;
+    }
+    amp-story h2 {
+      font-size: 36px;
+    }
+    amp-story p {
+      font-size: 16px;
+      line-height: 24px;
+    }
+    .bold {
+      font-weight: bold;
+    }
+    .bottom {
+      align-content: end;
+    }
+    .medium {
+      font-weight: 600;
+    }
+    .first {
+      padding-top: 65px;
+    }
+    .last {
+      padding-bottom: 65px;
+    }
+    .blue {
+      color: #4285F4;
+    }
+    .twenty-px {
+      font-size: 20px;
+    }
+    .center {
+      text-align: center;
+    }
+    .lh30 {
+      line-height: 30px;
+    }
+    .icon {
+      background-image: url(/img/AMP-Brand-White-Icon.svg);
+      background-repeat: no-repeat;
+      background-size: 50px 50px;
+      height: 50px;
+      object-fit: contain;
+      width: 50px;
+    }
+    .byline {
+      letter-spacing: 1.28px;
+      padding-bottom: 58px;
+    }
+    .introducing * {
+      line-height: 42px;
+    }
+    .subtitle-page {
+      padding-top: 80px;
+    }
+    .button {
+      align-items: center;
+      border: 4px solid #FFFFFF;
+      color: #FFFFFF;
+      display: flex;
+      height: 60px;
+      margin: 0 auto;
+      max-width: 240px;
+      text-decoration:none;
+    }
+    .button p {
+      font-size: 20px;
+      width: 100%;
+    }
+  </style>
+</head>
+  <body>
+    <amp-story standalone bookend-config-src="<%host%>/json/bookend.json">
+      <!-- ## Configuration -->
+      <!-- Include the `amp-story-auto-ads` tag on your page. -->
+      <!-- It's first child should be a `script` tag with a JSON configuration object
+      that contains details for how ads should be fetched and displayed.
+      `ad-attributes` is a map of key-value pairs, which correspond to the
+      attributes of the amp-ad element to be inserted.
+      -->
+
+      <amp-story-auto-ads>
+        <script type="application/json">
+          {
+              "ad-attributes": {
+                "type": "custom",
+                "data-url": "https://www.ampbyexample.com/json/amp-story-auto-ads"
+              }
+          }
+        </script>
+
+        <!-- After the script tag then inline any templates that you wish to include.
+        These templates will be populated from by a JSON response from the
+        server indicated by the `data-url` key above.
+        -->
+        <div>
+          <template type="amp-mustache" id="image-template">
+            <amp-img class="fill-img" layout="fill" src="{{imgSrc}}"></amp-img>
+          </template>
+
+          <template type="amp-mustache" id="video-template">
+            <amp-video autoplay loop
+              width="400"
+              height="750"
+              poster="{{poster}}"
+              layout="fill">
+              <source src="{{videoSrc}}" type="video/mp4">
+            </amp-video>
+          </template>
+        </div>
+      </amp-story-auto-ads>
+
```

Comment:
I assume you'd want to embed the story here. You should be able to declare the embed via amp-iframe inside the documentation block:

```
This is how the story looks....

<!---
<div class="abe-story-preview">
<amp-iframe media="(min-width: 650px)" src="/stories/introduction/amp_story_hello_world/preview/embed/"  width="440" height="732" sandbox="allow-scripts allow-popups allow-same-origin" ><div placeholder></div></amp-iframe>
</div>
-->
```

Reply:
I am having trouble getting this to work. Can you PTAL and tell me what I am doing wrong. (right now it is pointing at the existing story and I still can't get it to render)

Reply:
Sorry about that, you're hitting a few (non-documented) edge cases. This is what you have to do:

```
      <!-- See the full example below:
      <div class="abe-story-preview">
          <amp-iframe media="(min-width: 650px)"
              src="https://amp-by-example-api.appspot.com/amp-ads/amp_story_ads/amp-story-auto-ads/index.html"
              width="440" height="732" sandbox="allow-scripts allow-popups allow-same-origin">
            <div placeholder></div>
          </amp-iframe>
        </div> -->
      <!-- -->
```

The last `  <!-- -->` is needed to not show the following tag as a code section.